rllib/utils/test_utils.py

```diff
@@ -225,3 +225,21 @@ def check(x, y, decimals=5, atol=None, rtol=None, false=False):
                 if false is True:
                     assert False, \
                         "ERROR: x ({}) is the same as y ({})!".format(x, y)
+
+
+def check_learning_achieved(tune_results, min_reward):
```

Comment:
A call to this will be added to each example script and executed if an `--as-test` flag. It'll then check, whether some learning goal (reward) was reached in the example script. We currently do NOT test our example scripts for learning success.